app/assets/stylesheets/preview.sass

```diff
@@ -0,0 +1,3 @@
+/* Links on preview page. */
```

Comment:
While you're at that, add a linebreak after comments. Or get rid of them all, but just keep it consistent.